Dockerfile

```diff
@@ -8,13 +8,15 @@ MAINTAINER rpkish
 RUN apk update \
     && apk add --virtual build-dependencies build-base py3-lxml libxml2-dev libxslt-dev
 
+# run configuration steps
+COPY ./requirements.txt /usr/local/botiana/
+RUN pip install -r /usr/local/botiana/requirements.txt
+ 
 # Install botiana and copy modified settings and dictionaries 
-COPY ./botiana.py ./settings.py ./requirements.txt ./common.py ./keywords.py ./legacy_modules.py  \
```

Comment:
why add the second copy here?

Reply:
I split this because I wanted to quickly rebuild the image without waiting for the native compilation of one of the requirements. This allowed me to iterate on files and run with docker without waiting minutes for the build.

Reply:
makes sense.